src/app/payment/paymentMethod/HostedWidgetPaymentMethod.spec.tsx

```diff
@@ -183,6 +184,84 @@ describe('HostedWidgetPaymentMethod', () => {
         expect(component.isEmptyRender()).toBe(true);
     });
 
+    it('renders custom card component', () => {
```

Comment:
Hi @BC-AxelMalagon I just realize of this. HostedWidgetPaymentMethod shouldn't be aware of StripeV3 or any other payment-specific method. Please redesign this test to be agnostic.